pep/app/services/printer.ts

```diff
@@ -1,8 +1,23 @@
 import Service from '@ember/service';
 
+import Document from 'pep/pods/document/model';
 import printJS from 'print-js';
 
 export default class PrinterService extends Service {
+    /**
+     * Convert document data to a list of bibliographic items
+     *
+     * @param {Document[]} data
+     * @return {*}
+     * @memberof PrinterService
+     */
+    dataToBibliographicHTML(data: Document[]) {
+        const listItemsHtml = data.map((item) => {
+            return `<li class="mb-3 print-item">${item.authorMast}, ${item.year}, ${item.title}, ${item.documentRef}</li>`;
+        });
+        const html = `<div><ul class="list-unstyled">${listItemsHtml.join('')}</ul></div>`;
+        return html;
```

Comment:
probably not worth it yet, but would be nice if we could utilize the glimmer primitives/API, and be able to put this html in a .hbs template, and then load it and render it here w/glimmer.